styles.css

```diff
@@ -0,0 +1,458 @@
+@import url('https://fonts.googleapis.com/css2?family=Open+Sans:ital,wght@0,300..800;1,300..800&display=swap');
+@import url('https://fonts.cdnfonts.com/css/metropolis-2');
+
+* {
+    margin: 0;
+    padding: 0;
+    font-family: 'Metropolis', sans-serif;
+    color: #000;
+}
+
+header {
+    height: 75px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    border-bottom: 1px solid #ebf2f2;
+}
+
+header img {
```

Comment:
Please avoid using nested selectors. 
1 They are harder to override.
2 It is much easier to find style by single class name selector than by a combination of selectors